src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ReadClassifier.java

```diff
@@ -55,9 +56,10 @@ private static boolean hasInitialSoftClip( final List<CigarElement> cigarElement
         if ( firstEle.getOperator() == CigarOperator.HARD_CLIP && itr.hasNext() ) {
             firstEle = itr.next();
         }
+        final int clipStart = firstEle.getLength() - MIN_SOFT_CLIP_LEN;
         return firstEle.getOperator() == CigarOperator.SOFT_CLIP &&
-                firstEle.getLength() >= MIN_SOFT_CLIP_LEN &&
-                isHighQualityRegion(read.getBaseQualities(), 0);
+                clipStart >= 0 &&
+                isHighQualityRegion(read.getBaseQualities(), clipStart);
```

Comment:
Just to rephrase to verify that I understand what is happening: now we only check for low quality bases in the first "MIN_SOFT_CLIP_LEN" clipped bases as opposed to the whole soft-clipped region, and now we allow up to 3 low-quality bases in that region instead of zero. Is that correct? Is that the full substance of this change?